components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -157,31 +159,28 @@ func main() {
 	//
 	// Using map is intentional - we ensure that component name is not duplicated.
 	optionalComponentsDisablers := runtime.ComponentsDisablers{
-		"Kiali":   runtime.NewGenericComponentDisabler("kiali", "kyma-system"),
-		"Tracing": runtime.NewGenericComponentDisabler("tracing", "kyma-system"),
-		// TODO(workaround until #1049): following components should be always disabled and user should not be able to enable them in provisioning request. This implies following components cannot be specified under the plan schema definition.
-		"BackupInt":               runtime.NewGenericComponentDisabler("backup-init", "kyma-system"),
-		"Backup":                  runtime.NewGenericComponentDisabler("backup", "kyma-system"),
-		"KnativeEventingKafka":    runtime.NewGenericComponentDisabler(provisioning.KymaComponentNameKnativeEventingKafka, "knative-eventing"),
-		"KnativeProvisionerNatss": runtime.NewGenericComponentDisabler(provisioning.KymaComponentNameKnativeProvisionerNatss, "knative-eventing"),
-		"NatsStreaming":           runtime.NewGenericComponentDisabler(provisioning.KymaComponentNameNatsStreaming, "natss"),
+		components.Kiali:   runtime.NewGenericComponentDisabler(components.Kiali),
+		components.Tracing: runtime.NewGenericComponentDisabler(components.Tracing),
 	}
 	optComponentsSvc := runtime.NewOptionalComponentsService(optionalComponentsDisablers)
 
+	// Provides a list of disabled components per plan
```

Comment:
first of all, the constructor method name is pretty self explanatory, so we could remove this comment line. If you want to keep it, please move this above the method name in`disabled_components.go`:
```
//NewDisabledComponentsProvider does stuff
func NewDisabledComponentsProvider() DisabledComponentsProvider {
```

so it can be displayed in IDE hints pop-up

Reply:
removed it